tests/Apis/HelloWorld/HelloWorldCacheApi.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Weikio.ApiFramework.Abstractions;
+
+namespace HelloWorld
+{
+
+    public class HelloWorldCacheApi
+    {
+        private readonly IEndpointCache _cache;
+
+        public HelloWorldCacheApi(IEndpointCache cache)
+        {
+            _cache = cache;
+        }
+        private string GetTestString(string name)
+        {
+            return $"{name} (function)";
+        }
+
+        private async Task<string> GetTestStringAsync(string name)
+        {
+            return await Task.FromResult($"{name} (async function)");
+        }
+
+        private object GetTestObject(string name)
+        {
+            var testObject = new TestObject()
+            {
+                Name = name
+            };
+            return testObject;
+        }
```

Comment:
This is not used anywhere. Also `TestObject` will be unused if this method is removed.

Reply:
Removed.